eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -150,7 +151,8 @@ object LightningMessageCodecs {
       ("revocationBasepoint" | point) ::
       ("paymentBasepoint" | point) ::
       ("delayedPaymentBasepoint" | point) ::
-      ("firstPerCommitmentPoint" | point)).as[AcceptChannel]
+      ("firstPerCommitmentPoint" | point) ::
+      ("htlcBasepoint" | point)).as[AcceptChannel]
```

Comment:
field order has been changed in the specs PR:
    * [`33`:`revocation_basepoint`]
    * [`33`:`payment_basepoint`]
    * [`33`:`delayed_payment_basepoint`]
    * [`33`:`htlc_basepoint`]
    * [`33`:`first_per_commitment_point`]